newsfragments/2336.feature

```diff
@@ -0,0 +1 @@
+``dials.refine``: perform refinement of independent groups of experiments separately. The previous behaviour can be enforced using ``separate_independent_sets=False``
```

Comment:
Do you want to add a comment here to the effect that this is believed to not make a measurable difference to the quality of the results?